server/vertx-server/src/test/scala/sttp/tapir/server/vertx/VertxServerTest.scala

```diff
@@ -25,7 +26,35 @@ class VertxServerTest extends TestSuite {
           createServerTest,
           partContentTypeHeaderSupport = false, // README: doesn't seem supported but I may be wrong
           partOtherHeaderSupport = false
-        ).tests() ++ new ServerStreamingTests(createServerTest, VertxStreams).tests()
+        ).tests() ++ new ServerStreamingTests(createServerTest, VertxStreams).tests()++
+      (new ServerWebSocketTests(createServerTest, VertxStreams) {
+        override def functionToPipe[A, B](f: A => B): VertxStreams.Pipe[A, B] = in => new ReadStreamMapping(in,f)
+        override def emptyPipe[A, B]: VertxStreams.Pipe[A, B] = _ => new EmptyReadStream()
+      }).tests()
```

Comment:
adding tests the same way as for [zio](https://github.com/softwaremill/tapir/blob/5a1044c154f9bd1b629dfc567096f55408d93da3/server/vertx-server/zio1/src/test/scala/sttp/tapir/server/vertx/zio/ZioVertxServerTest.scala#L14-L37)